docs/docs/genai/tracing/integrations/listing/fireworksai.mdx

```diff
@@ -0,0 +1,307 @@
+For a list of available models see [the model library](https://fireworks.ai/models).
+
+# Tracing FireworksAI
+
+[MLflow Tracing](/genai/tracing) provides automatic tracing capability for FireworksAI through the OpenAI SDK compatibility. FireworksAI is [OpenAI SDK compatible](https://fireworks.ai/docs/tools-sdks/openai-compatibility#openai-compatibility), you can use the <APILink fn="mlflow.openai.autolog" /> function to enable automatic tracing. MLflow will capture traces for LLM invocations and log them to the active MLflow Experiment.
+
+## Quick Start
+
+```python
+import openai
+import mlflow
+
+# Enable auto-tracing
+mlflow.openai.autolog()
+
+# Create an OpenAI client configured for FireworksAI
+openai_client = openai.OpenAI(
+    base_url="https://api.fireworks.ai/inference/v1",
+    api_key="your-fireworks-api-key"
+)
+
+# Use the client as usual - traces will be automatically captured
+response = openai_client.chat.completions.create(
+    model="accounts/fireworks/models/deepseek-v3-0324", # For other models see: https://fireworks.ai/models
+    messages=[
+        {"role": "user", "content": "Why is open source better than closed source?"}
+    ],
+)
+```
+
+MLflow automatically captures the following information about FireworksAI calls:
+
+- Prompts and completion responses
+- Latencies
+- Model name
+- Additional metadata such as `temperature`, `max_completion_tokens`, if specified
+- Function calling if returned in the response
+- Any exception if raised
+
+:::tip
+For more information on FireworksAI's capabilities, visit the [FireworksAI documentation](https://fireworks.ai/docs).
+:::
+
+## Supported APIs
+
+Since FireworksAI is OpenAI SDK compatible, all APIs supported by MLflow's OpenAI integration work seamlessly with FireworksAI:
+
+#### Chat Completion API
+
+| Normal | Function Calling | Structured Outputs |   Streaming   |    Async     |
+| :----: | :--------------: | :----------------: | :-----------: | :----------: |
+|   ✅   |        ✅        |         ✅         |      ✅       |      ✅      |
+
+## Basic Example
+
+```python
+import openai
+import mlflow
+
+# Enable auto-tracing
+mlflow.openai.autolog()
+
+# Optional: Set a tracking URI and an experiment
+# If running locally you can start a server with:  `mlflow server --host 127.0.0.1 --port 5000`
+mlflow.set_tracking_uri("http://127.0.0.1:5000")
+mlflow.set_experiment("FireworksAI")
+
+# Configure OpenAI client for FireworksAI
+openai_client = openai.OpenAI(
+    base_url="https://api.fireworks.ai/inference/v1",
+    api_key="your-fireworks-api-key"
+)
+
+messages = [
+    {
+        "role": "user",
+        "content": "What is the capital of France?",
+    }
+]
+
+# To use different models check out the model library at: https://fireworks.ai/models
+response = openai_client.chat.completions.create(
+    model="accounts/fireworks/models/deepseek-v3-0324",
+    messages=messages,
+    max_completion_tokens=100,
+)
+```
+
+## Streaming
```

Comment:
We might want to have these code snippets of variations other than the simple example to be in WrappedTabs components to shorten the page a bit